scripts/aem.js

```diff
@@ -373,6 +373,53 @@ function decorateHorizontalRules(element) {
   });
 }
 
+/**
+ * Converts '<layout>' text nodes into containers to allow for nesting of blocks within layouts
+ * @param {Element} element container element
+ */
+function decorateLayouts(element) {
```

Comment:
You know we could make this file more readable by breaking all of these functions into partials and including them at the top of this file, then just running each function from this file.

Maybe we add a post-MVP ticket for AEM.js refactor, I'd love to do that clean up real quick!

Reply:
Let's add a post-MVP ticket for that!